storybookjs/storybook
# Ensure unique knob names across knob groups

Issue: #5391 

## What I did

Suffix knob names with the groupId, if provided (`${name}_${groupId}`).

## How to test

I updated `examples/official-storybook/stories/addon-knobs.stories.js` to have a knob with the same name in two different groups.

- Is this testable with Jest or Chromatic screenshots?
Yes
- Does this need a new example in the kitchen sink apps?
No
- Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
